README.md

```diff
@@ -1,4 +1,77 @@
-# hubrise-webclient
-A pet JS/HTML project to see new HubRise orders in a web page
+<a href="https://www.hubrise.com/"><img src="./screenshots/hubrise-logo.png" align="left" height="48"></a>
+<br><br><br>
 
-[Demo](http://philatist-fish-74534.netlify.com)
+# HubRise {{app_name}} Template
+
+{{app_name}} is an app that prints new orders received by your HubRise location, with just a few lines of code.
+
+You can fork this repository and make changes to the code to create your own app. 
+
+
+## Try the app
+
+First, you need to [create a HubRise account](https://manager.hubrise.com/signup). It's free and it only takes a minute. 
+
+Then, simply [open the app](http://philatist-fish-74534.netlify.com) in your browser. 
+
+You will be asked to connect the app to your HubRise account. 
+Granting permissions is a necessary step in every OAuth2.0 authentication process, and requires the user to manually **Allow** the app access. 
+
+![OAuth page granting access to the app](./screenshots/oauth-page.png)
+
+You will then see a single-page app that displays the list of the latest orders received by your HubRise location. 
+Each line in the logs displays the time, id, status, and total price of the order.
+
+![Single-page app with the orders from HubRise](./screenshots/logs-page.png)
+
+Clicking on one of the logs reveals additional information about the order, such as the customer's details and the purchased products. 
+
+![Modal with additional information about the order](./screenshots/single-order-modal.png)
+
+
+## What this app is NOT
+
+This app is not production ready! 
+Consider it some boilerplate code that you can buil upon to create your own app.
+It is also useful to show some of the things you should _not_ do in your app! 
+
+For example, in the first line of the `js/app.js` file, you will find the client secret. 
+This is bad coding practice. 
+
+Also, the user interface is too crude to be used in a real case scenario. 
+
+## Prerequisites
+
+...
+
+## Running {{app_name}} locally
```

Comment:
@antmonnier I guess someone might want to do this. Is it enough to un-comment the first lines in js/app.js to obtain this? I suspect not...

Reply:
Good idea!

Assuming the user has npm (or the more recent alternative) installed, he can use `serve`, a very lightweight JS server for static files.

Installation from the console:
`yarn global add serve`
or
`npm install -g serve`
(both commands might require root permissions)

At the root of the project, start the server by typing:
`serve`

And open the browser on the URL shown by the command, typically `http://localhost:5000`

Et voilà!

Reply:
It works! Witchcraft! 
The only thing I had to change was to add `sudo` before `npm`. I guess it does not hurt to add that.